### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by BraydanNewman**
Please check formatting, naming and logic @Bramers

### CODE REVIEW

File: CP1404/prac_05/word_occurrences.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from operator import itemgetter
+
+text = input("Text: ").split()
+
+words = {}
```
Comment (Bramers) [inline CP1404/prac_05/word_occurrences.py:5]: "words" sounds like it could be a list.

File: CP1404/prac_05/word_occurrences.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from operator import itemgetter
```
Comment (Bramers) [inline CP1404/prac_05/word_occurrences.py:1]: Itemgetter doesn't appear to be used.

File: CP1404/prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+for word in text:
+    if word in words:
+        words[word] += 1
+    else:
+        words[word] = 1
+
+max_length = max([len(str(word)) for word in words.keys()])
+
+for i in sorted(words.items()):
+    print(f"{i[0]:{max_length}}: {i[1]}")
```
Comment (Bramers) [inline CP1404/prac_05/word_occurrences.py:16]: Is there a better name than i to make this easier to understand?

**Review by Bramers — COMMENTED** [review]
Hey mate, Just a few small things I picked up, but besides that looks good. I like the way you did the word_occurrences program it's a lot more officiant than mine.

File: CP1404/prac_05/emails.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+user_completed = False
+
+emails = {}
+
+while not user_completed:
+    email = input("Email: ")
+    if email == "":
```
Comment (lindsaymarkward) [inline CP1404/prac_05/emails.py:7]: There's no reason to use this pattern, when the standard one is better.
Don't use `continue` to exit a loop; it's for continuing to the next iteration.

**Pull request merged**